### PR TITLE
[zh-cn]more glossaries translation

### PR DESCRIPTION
**Pull request by jasonren0403**
This PR proposed 1 changes and 4 new glossary translations towards simplified Chinese(zh-CN)
* New:
  1. Decryption
  2. Encryption
  3. Cipher
  4. Cipher suite
* Updated:
  1. Key

### CODE REVIEW

File: files/zh-cn/glossary/key/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: 密钥
+slug: Glossary/Key
+translation_of: Glossary/Key
+---
+密钥是一段用于 {{Glossary("cipher", "密码算法")}}{{Glossary("encryption","加密")}}和/或{{Glossary("decryption","解密")}}的信息。
+
+经加密的消息需要保证即使{{Glossary("cryptosystem","密码体制")}}中除了密钥外的其他部分在公开可见的情况下，依然保持安全性。
+
+在{{Glossary("symmetric-key cryptography", "对称密钥加密算法")}}中，加密与解密过程使用同样的密钥。在{{Glossary("public-key cryptography", "公钥加密算法")}}中，存在一对相互关联的 _公钥_ 和 _私钥_，公钥可以公开获取，私钥需要保密。公钥加密的消息，只有对应的私钥才能解密，反之亦然。
```
Comment (yin1999) [inline files/zh-cn/glossary/key/index.md:10]:
```suggestion
在{{Glossary("symmetric-key cryptography", "对称密钥加密算法")}}中，加密与解密过程使用同样的密钥。在{{Glossary("public-key cryptography", "公钥加密算法")}}中，存在一对相互关联的*公钥*和*私钥*，公钥可以公开获取，私钥需要保密。公钥加密的消息，只有对应的私钥才能解密，反之亦然。
```

**Review by yin1999 — COMMENTED** [review]
see: <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md>

**Review by yin1999 — APPROVED** [review]

**Comment by yin1999** [comment]
Thank you

File: files/zh-cn/glossary/cipher/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: 密码
+slug: Glossary/Cipher
+translation_of: Glossary/Cipher
+---
+在密码学（{{glossary("cryptography")}}）领域，**密码**是将{{glossary("Plaintext","明文")}}{{glossary("encryption", "编码")}}为不可读，且能够{{glossary("decryption", "解码")}}回原来的明文的算法。
+
+在信息时代很久以前，密码就已经很常见了（例如[替换密码](https://zh.wikipedia.org/wiki/替换式密码)和[移位密码](https://zh.wikipedia.org/zh-cn/古典密码#移位式密码)），但是它们之中除了[一次性密码本](https://zh.wikipedia.org/zh-cn/一次性密码本)以外，其他的均不满足密码学上的安全性。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/glossary/cipher/index.md:8]: 英文原文的后两个链接现在都重定向到一个页面去了，我选择保留了最接近的解法

[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/glossary/cipher/index.md:8]: Yes, I see

**Pull request merged**